docs/Getting-started.md

```diff
@@ -0,0 +1,52 @@
+# Start contributing to the Performance plugin
+This guide is intended to give details on how to contribute to the Performance plugin. From setting up the development environment to creating pull requests.
```

Comment:
I think those need to be connected since the latter is not a full sentence by itself. Maybe like this?
```suggestion
This guide focuses on how to contribute to the Performance plugin, from setting up the development environment to creating pull requests.
```